website/docs/ref/macro.md

```diff
@@ -4,20 +4,18 @@
 
 ## Installation
 
```

Comment:
Could you please add here a short paragraph saying that the user can choose between Babel macros or SWC Plugin? When I open the page currently, the SWC is at the bottom of my screen (and people with smaller screen sizes might need to scroll to see that we support SWC).

Also, it would be great to change the paragraph under the `Macros` heading to include that you don't need to use the Babel macros to use the `@lingui/macro` package.

Reply:
done